<?xml version="1.0"?>

<launch>
   <node pkg="cv_camera" type="cv_camera_node" name="cv_camera" output="screen">
        <param name="flip_image" value="false" />
        <param name="device_id" value="0" />
        <param name="image_width" value="320" />
        <param name="image_height" value="240" />
        <param name="frame_rate" value="25"/>
    </node>
    
  <node name="image_view" pkg="image_view" type="image_view" respawn="false" output="screen">
    <remap from="image" to="/cv_camera/image_raw"/>
    <param name="autosize" value="true" />
  </node>
</launch>

